clang/lib/Basic/Targets/PPC.cpp

```diff
@@ -782,6 +782,8 @@ ArrayRef<const char *> PPCTargetInfo::getGCCRegNames() const {
 const TargetInfo::GCCRegAlias PPCTargetInfo::GCCRegAliases[] = {
     // While some of these aliases do map to different registers
     // they still share the same register name.
+    // Strictly speaking, "ca" is a subregister of "xer". However currently we
```

Comment:
Should be a `FIXME:`?

Reply:
Maybe a note. I don't think we are modeling other fields like `SO`, `OV` in `XER` in the near future.

Reply:
The point of this patch was to allow the compiler to add "ca" as part of the clobber list. 

However, even if the compiler doesn't track other bits in the XER we may have to add them so that the compiler doesn't complain when a user tries to specify the bit. If that happens, would we have to add those other fields and then specify them as proper subregs of XER?

Also, which of these bits, if any, does GCC support on PowerPC?